package.json

```diff
@@ -106,9 +106,9 @@
     "swagger-ui": "2.1.4",
     "tv4": "^1.2.7",
     "velocity-animate": "^1.2.3",
-    "vue": "1.0.24",
+    "vue": "1.0.26",
     "vue-router": "0.7.13",
     "vue-strap": "^1.0.11",
-    "whatwg-fetch": "0.11.0"
+    "whatwg-fetch": "1.0.0"
```

Comment:
Beware here, the release process of `fetch` looks really… javascripty :) https://github.com/github/fetch/releases